src/screenshot.ts

```diff
@@ -0,0 +1,46 @@
+import { Page, Base64ScreenShotOptions } from "puppeteer";
+import { compile } from "handlebars";
+
+import { Options } from "./types";
+
+export async function makeScreenshot(
+  page: Page,
+  {
+    output,
+    type,
+    quality,
+    encoding,
+    content,
+    html,
+    selector,
+    beforeScreenshot,
+    transparent = false,
+    waitUntil = "networkidle0",
+  }: Options
+) {
+  let screenshotArgs: { quality?: number } = {};
+  if (type === "jpeg") {
+    screenshotArgs.quality = quality ? quality : 80;
+  }
```

Comment:
TS is able to infer the type directly if you want
```suggestion
  const screenshotArgs = (type === "jpeg") ? { quality: quality ?? 80 } : {}
```

Reply:
Fixed, thank you 🙏